src/images/falseColourPanel.component.tsx

```diff
@@ -55,6 +63,111 @@ interface FalseColourPanelProps extends FalseColourParams {
   changeUpperLevel: (value: number | undefined) => void;
 }
 
+function MyListSubheader(props: ListSubheaderProps) {
```

Comment:
What is the point of this custom component? Might as well just use `ListSubheader`